### ISSUE TITLE
[bug] different completions in comparison with codeium.vim

### ISSUE THREAD

**Issue by kaiphat** [Exafunction/codeium.nvim#117]
Codeium.nvim
1. Type `const fibo`. There aren't competions.
<img width="609" alt="image" src="https://github.com/Exafunction/codeium.nvim/assets/53742299/a16e658f-de97-4af9-8e7e-b0671cf4e87e">
2. Manually trigger cmp completion
<img width="609" alt="image" src="https://github.com/Exafunction/codeium.nvim/assets/53742299/a2d873d1-7197-4b3e-af81-35a25a7f0809">
3. Select first choice
<img width="609" alt="image" src="https://github.com/Exafunction/codeium.nvim/assets/53742299/b4314f0d-ef8a-48f0-b6c9-d7b2aadabd10">

Codeium.vim:
1. Just type `const fibo`. And great works when i continue typing.
<img width="609" alt="image" src="https://github.com/Exafunction/codeium.nvim/assets/53742299/fe245ebf-166b-44ad-91c0-b4d5f4482e14">

**Comment by pqn** [comment]
It does seem weird that `codeium.nvim` doesn't trigger a completion automatically. However, it's worth noting that there isn't determinism for what the server might return to you, for various reasons.

**Comment by kaiphat** [comment]
Yes, i understand, mb problem with cmp completion for several lines.

**Comment by gonviegas** [comment]
I use codeium.nvim and codeium.vim to have the best of both worlds:

![2023-12-02-133034_580x194_scrot](https://github.com/Exafunction/codeium.nvim/assets/47750371/a5be1e77-ead0-489a-bc7d-88491c62b52b)
![2023-12-02-133027_1005x316_scrot](https://github.com/Exafunction/codeium.nvim/assets/47750371/ad50c083-5245-460b-b9f2-f173127c80d0)

**Comment by kaiphat** [comment]
I'm glad to see that, but i checked your config and didn't find any special. Mb you are using paid version of codeium?

**Comment by gonviegas** [comment]
My config isn't updated, cant remember last time I did update it. 
I had the same problem as you regarding showing multi line completion and I know I did something to fix it but can't remember what...

**Comment by gonviegas** [comment]
My neovim config is updated now, check it out [here](https://github.com/gonviegas/nvim). 
Check the cmp config, might help you.

**Comment by russoalberto** [comment]
@kaiphat I had the same problem as you.

In your options.lua: https://github.com/kaiphat/dotfiles/blob/master/nvim/lua/options.lua

add `preview` in `o.completeopt = 'menuone,noselect'`

**Comment by kaiphat** [comment]
The same problem in copilot
https://github.com/zbirenbaum/copilot-cmp/issues/87

**Comment by gonviegas** [comment]
Then, the problem is definitely with your config, cmp most likely.

**Comment by gonviegas** [comment]
Files with a defined type may limit the output. 
Try to do ```const fibo``` in a file without its extension and check if the codeium suggestion is complete.

**Comment by cxwx** [comment]
still don't know how to complete with multi line
